sklearn/manifold/t_sne.py

```diff
@@ -387,10 +387,10 @@ def trustworthiness(X, X_embedded, n_neighbors=5, precomputed=False):
         T(k) = 1 - \frac{2}{nk (2n - 3k - 1)} \sum^n_{i=1}
             \sum_{j \in U^{(k)}_i} (r(i, j) - k)
 
-    where :math:`r(i, j)` is the rank of the embedded datapoint j
```

Comment:
I think there's some confusion in "according to", and r is not described as a function of i. Can you write it in your own words? Use multiple sentences. Perhaps describe U before you describe r. You can also say "for each sample i" to make it simpler

Reply:
I agree that `U` should be move before `r` and the "for each" should help defining things.
Perhaps you can use "input space" instead of original space and insist on the fact that the data sample `j` is the `r(i,j)-th` neighbors of the data sample `i`